src/LogNormal/LogNormalMath.sol

```diff
@@ -144,16 +143,15 @@ function computePriceGivenX(
     LogNormalParams memory params
 ) pure returns (uint256) {
     // $$\frac{1}{2} \sigma^2$$
```

Comment:
Maybe we can remove that comment since we're using a dedicated function? (Let's move the formula to the NatSpec of that function if it's not done already)